src/Sylius/Bundle/AttributeBundle/spec/Form/EventSubscriber/BuildAttributeValueFormSubscriberSpec.php

```diff
@@ -89,7 +95,9 @@ function it_adds_a_value_form_field_with_correct_type_based_on_the_attribute_id(
         $attributeRepository,
         AttributeInterface $attribute,
         Form $form,
-        FormEvent $event
+        FormEvent $event,
+        $attributeValueTypeConfigurationFactory,
```

Comment:
If you don't use typehints for attributes from constructor, it would be better to put them at the beginning